api/src/test/java/org/pmiops/workbench/genomics/GenomicExtractionServiceTest.java

```diff
@@ -124,7 +125,6 @@ public class GenomicExtractionServiceTest {
     GenomicExtractionService.class,
     WorkspaceAuthService.class,
   })
-  @MockBean({InitialCreditsService.class})
```

Comment:
supposedly this can be replaced by `@MockitoBean(types = {InitialCreditsService.class})` but it didn't work for me